docs/src/main/sphinx/release/release-479.md

```diff
@@ -0,0 +1,149 @@
+# Release 479 (TBD 2025)
+
+## General
+
+* Support automatic TLS certificate generation in the `ANNOUNCE` node discovery mode. ({issue}`27030`)
+* Add support for specifying the location of the Exchange Manager configuration file. ({issue}`26611`)
+* Publish query finishing time in event listener. ({issue}`27202`)
+* Add lineage support for output columns in all SELECT queries ({issue}`26241`)
+* Require JDK 25 to build and run Trino ({issue}`27153`)
+* Require the JDK Vector API to be enabled at runtime ({issue}`27340`)
+* Add support for `SET DEFAULT` and `DROP DEFAULT` statements. ({issue}`26162`)
+* Add `array_first` and `array_last` functions. ({issue}`27295`)
+* Enhance after `RemoveTrivialFilters` call `RemoveEmptyUnionBranches` to prune empty union branches. ({issue}`21506`)
```

Comment:
This does not convey what's the end user visible change. 
Also link might is wrong. Is this #26349? There is another bullet below also linking to #21506, related to #27191 PR.

Reply:
It's linking to `21506` because it fixes it. should we put also pr number here as well?

the release notes should be: Fix query failure when `UNION` involve empty values branch ? @kasiafi @Max-Cheng

Reply:
```
* Fix query failure when one of the branches of a `UNION` is known to produce zero rows during query planning.
```